src/shadowbox/server/api.yml

```diff
@@ -139,6 +139,13 @@ paths:
       description: Creates a new access key
       tags:
         - Access Key
+      parameters:
+        - name: encryptionMethod
+          in: path
```

Comment:
This is not being passed in the request path. Could you fix it?
I think the right spec it to specify the `requestBody` here, like in the `/name` method above.

Spec: https://swagger.io/specification/

Reply:
> This is not being passed in the request path. Could you fix it? I think the right spec it to specify the `requestBody` here, like in the `/name` method above.
> 
> Spec: https://swagger.io/specification/

done.

Reply:
PTAL